app/config/services.yml

```diff
@@ -277,6 +277,15 @@ services:
         tags:
           - name: security.voter
 
+    elife.journal.security.voter.feature.wallenberg:
+        class: eLife\Journal\Security\Voter\TimeVoter
+        public: false
+        arguments:
+          - FEATURE_WALLENBERG
+          - "@=strtotime('2018-01-01T00:00:00Z')"
+        tags:
+          - name: security.voter
```

Comment:
people would probably like to see the modified version before it goes online, so another voter based on a query string parameter would enable them to do so

Reply:
/me awaits requirement